test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj

```diff
@@ -40,7 +40,7 @@
   "./target/master-var/log")
 
 (def ca-dir
-  "./target/ca")
+  "./target/master-conf/ca")
```

Comment:
At the moment, this is passing tests for me locally. I'm very skeptical, and think that @justinstoller might be right that there's another piece to this that I've overlooked.